app/visualisation_methods/lindev/plugin.rb

```diff
@@ -7,7 +7,7 @@ def handler
     if parameters["id"] && parameters["param_x"] && parameters["param_y"]
```

Comment:
To trochę starszy kod - ale czy parameters wcześniej jest jakoś walidowane - np. czy parametry domyślnie są konwertowane na stringi? Jeśli nie - konwertować tutaj, czyli np. parameters["param_x"] = parameters["param_x"].to_s